js/func.js

```diff
@@ -0,0 +1,453 @@
+"use strict";
```

Comment:
Why this file here?

Reply:
when we create a .ts file and run, it creates a JS file on the same path of TS file.